### PR TITLE
Overview patch

### PR DESCRIPTION
**Pull request by cmharlow**
@guegueng @AudreyAltman your emails are in the updated overview - they're on the dpla site, but want to make sure this is okay here too.

i've added install recs and was going to add some tests for setup and screenshots (walking over to our windows lab now since i don't have a windows lab).

hope this looks okay, and will open a new patch branch for expanding the workshop portions - including a list of requested topics covered in the bash/unix shell portion for sake of the python workshop.

### CODE REVIEW

**Comment by cmharlow** [comment]
Also, quickly - the Git/GitHub requirement is really just for ease of sharing the materials the day of. If folks don't show up with Git + GitHub ready, I'll have some thumbdrives. Also, going to add download + review of the files to the proposed bash line up.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,86 @@
...
-*	Command line basics: Audrey (30)
-*	Harvesting OAI feed at the command line: Christina (30) 
-*	break (15)
-*	Metadata analysis: Christina (90 - includes working time)
-*	Other tools to explore: Gretchen (30)
+## About the Workshop
 
-# Requirements:
-(more to come)
+The DPLAfest 2017 DPLA Metadata Analysis Bootcamp for Hubs & Partners will be held **1 PM to 5 PM, Wednesday, April 19, 2017 at the Chicago Public Library’s Harold Washington Library Center** (enter at 401 S. Plymouth Street and read more about space logistics [here](https://dp.la/info/get-involved/dplafest/april-2017/travel/)). Exact room number TBA (we will update this repository and email registered participants).
```
Comment (cmharlow) [inline README.md]: Need to confirm address + room number here (presuming it is in the same building as the rest of the DPLAfest).

**Comment by cmharlow** [comment]
updated PR overview-patch branch with installation check tests, images/screenshots, and some links to move around the overview document. Removed the slack messages as a communication method bc it felt a bit overkill.

**Comment by guegueng** [comment]
It seems fine to me. Are you sure that any version of Python after 2.7 will work? I was unable to get the last version of the code working with 3.6, but maybe you'd updated it?

**Comment by cmharlow** [comment]
any version of python 2.x.x after 2.7

**Comment by cmharlow** [comment]
depending on which analysis script you're running, it can work with python 3, but i'm not promising that will be done for all of the scripts here

**Comment by cmharlow** [comment]
Updated per @guegueng's comment - gonna merge bc otherwise seems to be okay.

**Pull request merged**